internal/pkg/v2/utils/address.go

```diff
@@ -0,0 +1,121 @@
+//
```

Comment:
rename this file to restaddress.go, because this is no general address utils

Reply:
Renamed.